### ISSUE TITLE
option for sub brunches

### ISSUE THREAD

**Issue by laslappas** [vipzhicheng/logseq-plugin-lock#11]
It'll be great and useful to have the option to encrypt a block when all his sub blocks

**Comment by jaroet** [comment]
Yes, I would like to see that too! When locked (maybe a separate function) all nested content below the block you selected will be encrypted. 

Because it is currently only for the current block it is only usable for passwords or one-sentence blocks. 

My usecase will be that for instance a certain log of a conversation is privacy sensitive I can use all of Logseq's functionality with blocks, links, references, ect and have then everything be encrypted in one go. 

Thanks for considering ...

Regards,
Jeroen
